src/SourceBuild/content/test/Directory.Build.props

```diff
@@ -1,10 +1,2 @@
 <Project>
```

Comment:
This file shouldn't exist as want to import the root Directory.Build.props and targets file.